client/my-sites/upgrades/checkout-thank-you/google-apps-details.jsx

```diff
@@ -27,6 +27,7 @@ const GoogleAppsDetails = () => {
 			buttonText={ i18n.translate( 'Learn more about Google Apps' ) }
 			href={ supportUrls.GOOGLE_APPS_LEARNING_CENTER }
 			target="_blank"
+			rel="noopener noreferrer"
```

Comment:
`<PurchaseDetail />` explicitly detects and passes along `target`, but not `rel`